### ISSUE TITLE
Can I install it on Debian OS?

### ISSUE THREAD

**Issue by calex80** [Dewalt-arch/pimpmykali#38]
I would like to install it on debian OS, Does it possible?

**Comment by BrettBass** [comment]
Its runs a menu, pick and choose what you'd like.

**Comment by calex80** [comment]
Yes, that's what I expected, but says `[!!] Kali Linux Not Detected - WSL/WSL2/Anything else is unsupported [!!] `
![image](https://github.com/Dewalt-arch/pimpmykali/assets/58860205/8ac7768f-4232-4bc8-9ea7-a0c722c661aa)

**Comment by BrettBass** [comment]
You could delete lines 106 - 109. You'll get a lot of warnings about symlinks as I just tested it in an Arch VM, I recommend installing Kali on a flash drive and running this there if you want to enjoy the features. 
![image](https://github.com/Dewalt-arch/pimpmykali/assets/62232561/66666627-8420-4ffa-b413-cc23c49dd1bb)
![image](https://github.com/Dewalt-arch/pimpmykali/assets/62232561/a90b0758-9606-4c80-b446-81b9ab0ba004)

**Comment by BrettBass** [comment]
You could also stick to the stand-alone functions 
![image](https://github.com/Dewalt-arch/pimpmykali/assets/62232561/646c6ac8-5963-4f22-95ed-26abd781df95)

**Comment by calex80** [comment]
Thank you a lot, its working
